src/api/flaskr/service/profile/profile_manage.py

```
@@ -0,0 +1,639 @@
+        profile_item.profile_check_model = profile_check_model
+        profile_item.profile_check_model_args = str(profile_check_model_args)
+        profile_item.updated_by = user_id
+        if type == PROFILE_TYPE_INPUT_TEXT and not profile_prompt:
+            raise_error("PROFILE.PROMPT_REQUIRED")
+        if type == PROFILE_TYPE_INPUT_SELECT:
+            if len(items) == 0:
+                raise_error("PROFILE.ITEMS_REQUIRED")
+            profile_item_value = ProfileItemValue.query.filter_by(
+                profile_id=profile_id, status=1
+            ).all()
+            for profile_item_value in profile_item_value:
```

Ruff (0.8.2)

Loop control variable `profile_item_value` overrides iterable it iterates

(B020)